### PR TITLE
SAAP-69 Adding AWS Cloud Provider requirements

### PR DESCRIPTION
**Pull request by TehreemNisa**
SAAP-69 Adding AWS Cloud Provider requirements

### CODE REVIEW

**Comment by stakater-user** [comment]
@TehreemNisa Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/89f7c8d5d52d07a983e4b2af184590d05a53499c/checks) has Failed!

File: content/sre/cloud-provider/aws.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
 # AWS
 
-_TBD_
+Stakater Agility Platform supports provisioning clusters on AWS.
+
+## Subscription
+
+An AWS subscription is needed to create and manage cluster on azure. The following criteria must be met
```
Comment (usamaahmadkhan) [inline content/sre/cloud-provider/aws.md]: AWS Account not subscription. Azure has subscription. Plz change throughout the document.

File: content/sre/cloud-provider/aws.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+
+- Stakater shall be provided access to a separate subscription.
+- The Stakater user must have admin access on the created subscription.
+- Stakater shall share an email address that should have full access to this subscription.
+- The customer must not use this subscription for other purposes and only stakater shall be responsible for managing resources created in this subscription.
+- Resource limits must be applied on the subscription and only the following resources must be allowed to be created.
+
+  |Type        | Limit |
+    |------------|------------|
+  | Virtual Machines | The limit should be 12 i.e of 3 worker + 3 master |
```
Comment (usamaahmadkhan) [inline content/sre/cloud-provider/aws.md]: `Varies. The limit should be 12 initially. (Initial deployemnt is 3 master + 3 infra + 3 worker)`

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

File: content/sre/cloud-provider/aws.md (hunk 1 above)
Comment (TehreemNisa) [reply content/sre/cloud-provider/aws.md]: done

File: content/sre/cloud-provider/aws.md (hunk 2 above)
Comment (TehreemNisa) [reply content/sre/cloud-provider/aws.md]: done

**Comment by stakater-user** [comment]
@TehreemNisa Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/f75459cd20156eacffa8d7f22d9e0d7a8020b09a/checks) has Failed!

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-137-99837208`

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.83`

**Pull request merged**